package.json

```diff
@@ -198,7 +198,7 @@
   ],
   "gxVersion": "0.12.1",
   "language": "go",
-  "license": "",
+  "license": "(MIT OR APACHE)",
```

Comment:
should be `Apache 2` I believe

Reply:
https://spdx.org/licenses/ says Apache-2.0. Changed!

Reply:
perfect, thank you